.github/workflows/pull-request-checks.yaml

```diff
@@ -100,3 +100,45 @@ jobs:
         run: make -C regression/verilog test
       - name: Print ccache stats
         run: ccache -s
+
+  # This job takes approximately 15 minutes
+  check-centos8-make-gcc:
+    name: CentOS 8
+    runs-on: ubuntu-20.04
+    container:
+      image: centos:8
+    steps:
+      - name: Install Packages
+        run: |
+          sed -i -e "s|mirrorlist=|#mirrorlist=|g" -e "s|#baseurl=http://mirror.centos.org|baseurl=http://vault.centos.org|g" /etc/yum.repos.d/CentOS-Linux-*
+          yum -y install make gcc-c++ flex bison git rpmdevtools wget zlib-devel
+          wget --no-verbose https://github.com/ccache/ccache/releases/download/v4.8.3/ccache-4.8.3-linux-x86_64.tar.xz
+          unxz ccache-4.8.3-linux-x86_64.tar.xz
+          tar xf ccache-4.8.3-linux-x86_64.tar 
```

Comment:
Use `tar xJf` instead of the separate `unxz` step?

Reply:
done